apps/estuary/lib/estuary/event_table.ex

```diff
@@ -9,6 +9,13 @@ defmodule Estuary.EventTable do
     |> Prestige.prefetch()
   end
 
+  def insert_event(author, create_ts, data, type) do
+    Prestige.execute("INSERT INTO #{table_name()}
+      (author, create_ts, data, type)
+      values ('#{author}', #{create_ts}, '#{data}', '#{type}')")
+    |> Prestige.prefetch()
```

Comment:
Do we want to handle failures to write to presto, such as presto down by dlq'ing messages that invoke presto errors?  I feel like other apps dont do this?  Wondering what the rest of the team thinks.

Reply:
Yes, that will help in logging the insert failure early.

Reply:
Agree, might be worth patterning matching out an `:error` case

Reply:
IF presto is down we don't want to DLQ. We want the app to crash / retry / reveal that it is missing the infrastructure it needs. If a message is bad/ we have an error with a single message we want to dlq that message.